app/views/govuk_publishing_components/components/_document_list.html.erb

```diff
@@ -33,7 +33,8 @@
               item[:link][:path],
               data: item[:link][:data_attributes],
               class: "#{item_classes} gem-c-document-list__item-link",
-              lang: item[:link][:locale].presence
+              lang: item[:link][:locale].presence,
+              rel: item[:link][:rel].presence,
```

Comment:
Are we happy to allow anything to be passed in here? We might be, I'm uncertain.

Reply:
That's a good point. I don't think there are any use cases for anything other than `external` right now. Would it be valuable to replace this with just an external attribute? Eg: `external: true`. which would render the `rel` attribute.

Reply:
I don't know what else we might be using but I looked up the list and there's a few that might be useful, e.g. nofollow.

I was thinking something like this https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_fieldset.html.erb#L8

Reply:
That's probably a bit more extensible. So here's a proposal: I implement that fieldset solution and do one of the following:

1. Just include `external` for now
2. Include everything in that table that the `a` tag specifies as an annotation

Reply:
Sounds good.